src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Class to execute bye command.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public ByeCommand(String inputCommand, boolean isExit) {
+        super(inputCommand, isExit);
+    }
+
+    /**
+     * Execute the given task.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). There should also be an empty line between description and parameter section.